### PR TITLE
Add Shadow document explaining what RT shadows are, what they do, and…

### PR DESCRIPTION
**Pull request by jberkus**
… how the shadow program is expected to run.

We haven't had a doc that explains what shadowing is.  This doc also outlines what general kinds of activities shadows do, what they can expect from Leads, and what happens after the release.

/kind documentation
/assign @justaugustus 

cc:
@lachie83 @jdumars @guineveresaenger @claurence 

ref: 
#688 #670

### CODE REVIEW

**Comment by jberkus** [comment]
/assign @claurence 

Holding until both Stephen & Claire look over it:

/hold

File: release-team/shadows.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+## Week 1
+
+The first week should be used to get Shadows set up with all of the "paperwork" they need in order to participate fully in the Release Team:
+
+* Sign the [CNCF CLA](https://github.com/kubernetes/community/blob/master/CLA.md), which they need for all contributions.
+* Apply to be an Org Member if they are not already (see below).
+* Join the [#sig-release slack channel](https://slack.k8s.io/).
+* Get added to the `kubernetes-release-team` group by the Release Lead.
+* Join the [sig-release](https://groups.google.com/forum/#!forum/kubernetes-sig-release) and [release-team](https://groups.google.com/forum/#!forum/kubernetes-release-team) Google Groups and calendars.
+* All all contact information to their Release Team's contact sheet.
```
Comment (claurence) [inline release-team/shadows.md]: I think either there is a duplicate "all" or one "all was perhaps meant to be an "add"?

File: release-team/shadows.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+For the "main" part of the development cycle, new shadows should be learning their roles and starting to take on tasks for their roles. This may include:
+
+* Attending the weekly Release Team meeting, and possibly delivering the report for their Role for the week.
+* Checking on daily statuses of things in their roles and recording them in documents shared by the team.
+* Contacting and sending reminders to SIGs.
+* Reviewing and resolving open issues in sig-release and test-infra that affect their roles, especially ones left over from last release.
+* Other tasks as defined in the [Role Handbook](https://github.com/kubernetes/sig-release/tree/master/release-team).
+
+For a couple roles, particularly Enhancements, this first period is actually the busiest period of the release, so there will be a lot for the Shadows to do.  For other roles, the shadows for a role should expect to be in communication with their Lead and each other a few times a week, via Slack, shared documents, and Github issues/project boards.
```
Comment (claurence) [inline release-team/shadows.md]: maybe worth adding something about asking the role lead when is the bushiest time of the release cycle as something shadows should do in the early weeks?

File: release-team/shadows.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+**Becoming Role Lead**: The main goal of the Shadow program is to train new Leads.  So, if you are ready to take on Leading the Role in the succeeding cycle, then let your Role Lead know.  They will be selecting a nominee or nominees to recommend to the incoming Release Lead, who has the final decision.  You can also skip a release cycle, and volunteer to become Role Lead in a later release cycle, in which case you should contact the incoming Release Lead as soon as they are announced.  There is usually an issue in the SIG-Release repository called "Assemble Kubernetes X.YY Release Team" where you can comment with your interest in a Role.
+
+**Another role in SIG-Release**: some Shadows go on to do something else within SIG-Release.  For example, they may join the Branch Managers as an RMA, or they may decide to take on a release-related project that's not directly tied to the release cycle.  If you are interested in the Branch Managers, wait for them to announce their next recruitment cycle, or ask in the SIG-Release Slack channel.
```
Comment (claurence) [inline release-team/shadows.md]: what does RMA stand for?

File: release-team/shadows.md (hunk 2 above)
Comment (jberkus) [reply release-team/shadows.md]: better, say "consult the handbook". will add.

File: release-team/shadows.md (hunk 3 above)
Comment (jberkus) [reply release-team/shadows.md]: Release Management Associate.  Don't blame me, @justaugustus came up with the name.  ;-)

Comment (claurence) [reply release-team/shadows.md]: I see - what are y'alls thoughts on spelling out the acronym here so new folks who aren't familiar with the term know what it means ?

Comment (jberkus) [reply release-team/shadows.md]: added to list of changes for next commit.

**Comment by jberkus** [comment]
All of @claurence's feedback has been incorporated.

**Pull request merged**